Character_Builder/models.py

```diff
@@ -35,14 +35,53 @@
 # As this is in the character builder folder, this will focus on
 # the character information 
 
+# Automatically creates CharacterIDs
+def newCharacterID():
+    previousCharacter = Character.objects.last()
+    if not previousCharacter is None:
+        newID = previousCharacter.characterID + 1
+    else:
+        newID = 0
+
+    return newID
+
+# Automatically creates AbilityScoreSetIDs
+def newAbilityScoreSetID():
+    previousAbilityScoreSet = AbilityScoreSet.objects.last()
+    if not previousAbilityScoreSet is None:
+        newID = previousAbilityScoreSet.abilityScoreSetID + 1
+    else:
+        newID = 0
+
+    return newID
+
+# finds a default user
+def defaultUser():
+    default = User.objects.first()
+
+    if default is None:
+        default = User.objects.create_user('defaultUser', password='djangoproject')
+
+    return default
+
+# Automatically creates abililtyScoreSetIDs
+def newAbilityScoreSetID():
```

Comment:
Is there a typo here? Should this be newCharacterID() instead of newAbilityScoreSetID()?

That said, as Ben said, you could replace this with a primary key with:
models.AutoField(primary_key=True)